### PR TITLE
Ruby programs

### PR DESCRIPTION
**Pull request by anushakantubuktha**

### CODE REVIEW

File: c.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+print "enter  first number  : "
+n1 = gets.chomp.to_i
```
Comment (vsandeepsrivastav) [inline c.rb]: hey @anushakantubuktha can you rename the files as anusha_c, anusha_prime etc and upload the again?

**Comment by anushakantubuktha** [comment]
Ok Sir

File: c.rb (hunk 1 above)
Comment (vsandeepsrivastav) [inline c.rb]: delete the old files

Comment (anushakantubuktha) [inline c.rb]: ok sir

**Pull request merged**